CONTRIBUTING.md

```diff
@@ -426,6 +432,14 @@ To become a Code Owner, open a PR with the following changes:
 
 Be sure to tag the existing Code Owners, if any, within the PR to ensure they receive a notification.
 
+### Emeritus roles
+
+Contributors that are stepping down or reducing their role in the project are encouraged to move to [emeritus](https://github.com/open-telemetry/community/blob/main/guides/contributor/membership.md#emeritus-maintainerapprovertriager). 
+
+In the event that a contributor steps down without prior notice, the maintainers will attempt to contact the contributor via both Github and the CNCF Slack to confirm their status. After two weeks, if the contributor is an approver or maintainer, they may be removed from the Github review auto-assignment. If the contributor does not respond within a period of one month, they may be moved to emeritus status at the discretion of the maintainers.
+
+In case of disagreement, this will be submitted to a majority vote among the maintainers (possibly excluding the contributor in question).
```

Comment:
Can we define who disagrees about what?

Reply:
I am torn about this, maybe we should just say that we must make a majority vote, that would eliminate some ambiguity and possibility for bias

Reply:
Reworded as discussed in the SIG meeting